### PR TITLE
aerc: update to 0.9.0

### PR DESCRIPTION
**Pull request by shtayerc**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

File: srcpkgs/aerc/template [hunk 1]
```diff
@@ -10,13 +10,14 @@ maintainer="shtayerc <david.murko@mailbox.org>"
 license="MIT"
 homepage="https://aerc-mail.org"
 distfiles="https://git.sr.ht/~rjarry/aerc/archive/${version}.tar.gz"
-checksum=e149236623c103c8526b1f872b4e630e67f15be98ac604c0ea0186054dbef0cc
+checksum=5bea713b211dcd85599c697a0f6563e07e2c04fd1443416545eea0a9a83a4f8d
 
 do_configure() {
```
[the remark this answers is not in the document]
Comment (shtayerc) [reply srcpkgs/aerc/template]: You are right. Done

**Comment by novakne** [comment]
8.2 have been released https://git.sr.ht/~rjarry/aerc/refs/0.8.2

File: srcpkgs/aerc/template [hunk 2]
```diff
@@ -1,29 +1,33 @@
 # Template file for 'aerc'
 pkgname=aerc
-version=0.7.1
+version=0.8.2
 revision=1
 build_style=go
+go_import_path="git.sr.ht/~rjarry/aerc"
+go_build_tags=notmuch
+go_ldflags="-X main.Version=${version}"
 hostmakedepends="scdoc git"
```
Comment (paper42) [inline srcpkgs/aerc/template:9]: git should not be needed

**Comment by novakne** [comment]
0.9.0 have been released https://git.sr.ht/~rjarry/aerc/refs/0.9.0

File: srcpkgs/aerc/template [hunk 3]
```diff
@@ -1,29 +1,33 @@
...
+	vmkdir usr/share/aerc
+	vcopy config/*.conf usr/share/aerc
+	vcopy filters usr/share/aerc
+	vcopy templates usr/share/aerc
+	vinstall config/default_styleset 644 usr/share/aerc/stylesets default
 
-do_install() {
-	make STRIP=true PREFIX=/usr DESTDIR=${DESTDIR} install
-}
+	vinstall contrib/aerc.desktop 644 usr/share/applications
```
Comment (paper42) [inline srcpkgs/aerc/template:30]: This is not really maintainable. Would something like this work? `do_install() { make install }`

Comment (shtayerc) [reply srcpkgs/aerc/template:30]: We would have to patch Makefile then. Just running `make install` starts building aerc again.

Comment (shtayerc) [reply srcpkgs/aerc/template]: @paper42 Done.

Comment (paper42) [reply srcpkgs/aerc/template]: Hmm, that's unfortunate, do you think we could convince upstream to have a install-data make target? Then we could drop the patch and just set `make_install_target="install-data"`

Comment (shtayerc) [reply srcpkgs/aerc/template]: I think not. We are not compiling in the recommended way (using make).

**Pull request merged**